src/utils/load-image.tsx

```diff
@@ -29,18 +29,29 @@ export function* createImageDef(world: HubsWorld, url: string, contentType: stri
     ratio,
     projection: ProjectionMode.FLAT,
     alphaMode,
+    alphaCutoff: 0.5,
     cacheKey
   };
 }
 
-export function* loadImage(world: HubsWorld, url: string, contentType: string) {
+export function* loadImage(world: HubsWorld, eid: EntityID, url: string, contentType: string) {
   const imageDef = yield* createImageDef(world, url, contentType);
 
-  return renderAsEntity(
-    world,
-    <entity
-      name="Image"
-      image={imageDef}
-    />
-  );
+  if (MediaImage.projection[eid]) {
+    imageDef.projection = APP.getString(MediaImage.projection[eid]) as ProjectionMode;
+  } else {
+    imageDef.projection = ProjectionMode.FLAT;
+  }
+  if (MediaImage.alphaMode[eid]) {
+    imageDef.alphaMode = APP.getString(MediaImage.alphaMode[eid]) as AlphaMode;
+  } else {
+    imageDef.alphaMode = AlphaMode.Opaque;
+  }
+  if (MediaImage.alphaCutoff[eid]) {
+    imageDef.alphaCutoff = MediaImage.alphaCutoff[eid];
+  } else {
+    imageDef.alphaCutoff = 0.5;
+  }
```

Comment:
I don't really get these `MediaImage` properties checking are for. An entity has `MediaImage` component but those properties are not set? If so, how does it happen?

Or, checking whether an entity has `MediaImage` component? If so, I don't think accessing component properties that is not assigned to an entity is safe. `hasComponent()` should be used instead.

Reply:
See https://github.com/mozilla/hubs/pull/6111#discussion_r1288496997